backend/segmentation/models/microtubule/segment_mt.py

```
@@ -73,14 +77,129 @@ def _normalize(img: np.ndarray) -> np.ndarray:
+    process, so a memo would buy nothing while forcing every test that patches
+    ``pathlib`` to remember to invalidate it.
+
+    A missing attribute is a bug in the caller's table, not a fact about the
+    host, so it propagates instead of being flattened into ``False`` (which
+    would silently remap on a platform where the class works fine).
+    """
+    cls = getattr(sys.modules[module], name)
+    try:
+        cls(".")
+        return True
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)